src/components/SidebarPlugin/SideBar.vue

```diff
@@ -10,7 +10,7 @@
               <i class="ni ni-compass-04"></i> SSO Tools
             </router-link>
 
-            <slot name="mobile-right">
+            <!-- <slot name="mobile-right">
```

Comment:
@MurthyAvanithsa , lets delete this if not reequired

Reply:
@venusripada Ok